[Bug]: World Crash When Using Splash Potion of Mana Near Other Entities

### Describe the bug

When I used a splash potion of instant mana near other entities that don't have the max_mana attribute, my world crashed. It still crashes when I try to rejoin into the world too. I have version 1.18.2-1.2.2, there doesnt seem to be an option of the version in the version dropdown.

### Steps to Reproduce

1. Craft a splash potion of mana
2. Use the splash potion near other entities without the max_mana atrribute (sheep, chickens, cows, ect.)

### Expected behavior

Player to receive the effects of the potion while the potion effect would not effect other entities.

### Screenshots

_No response_

### Version

1.18.2-1.2.0

### Forge Version

40.1.86

### Other Mods

![Screenshot 2022-12-28 171129](https://user-images.githubusercontent.com/81721136/209877416-a7c89389-3dc6-4051-adbe-b929112fde53.png)


### Log

```shell
net.minecraft.ReportedException: Ticking entity
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:906) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:84) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Can't find attribute arsmagicalegacy:max_mana
	at net.minecraft.world.entity.ai.attributes.AttributeSupplier.m_22260_(AttributeSupplier.java:21) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.entity.ai.attributes.AttributeSupplier.m_22245_(AttributeSupplier.java:28) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.entity.ai.attributes.AttributeMap.m_22181_(AttributeMap.java:62) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.entity.LivingEntity.m_21133_(LivingEntity.java:1818) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at com.github.minecraftschurlimods.arsmagicalegacy.common.magic.ManaHelper.getMaxMana(ManaHelper.java:65) ~[arsmagicalegacy-1.18.2-1.2.2.jar%2356!/:1.2.2]
	at com.github.minecraftschurlimods.arsmagicalegacy.common.effect.InstantManaEffect.m_19461_(InstantManaEffect.java:19) ~[arsmagicalegacy-1.18.2-1.2.2.jar%2356!/:1.2.2]
	at net.minecraft.world.entity.projectile.ThrownPotion.m_37547_(ThrownPotion.java:152) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.entity.projectile.ThrownPotion.m_6532_(ThrownPotion.java:105) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.entity.projectile.ThrowableProjectile.m_8119_(ThrowableProjectile.java:62) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.level.ServerLevel.m_8647_(ServerLevel.java:652) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.level.Level.m_46653_(Level.java:487) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.level.ServerLevel.m_184063_(ServerLevel.java:319) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.world.level.entity.EntityTickList.m_156910_(EntityTickList.java:54) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:299) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:902) ~[client-1.18.2-20220404.173914-srg.jar%23140!/:?]
	... 5 more
```

--- a/src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/effect/InstantManaEffect.java
+++ b/src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/effect/InstantManaEffect.java
@@ -2,6 +2,7 @@ package com.github.minecraftschurlimods.arsmagicalegacy.common.effect;
 
 import com.github.minecraftschurlimods.arsmagicalegacy.api.ArsMagicaAPI;
 import com.github.minecraftschurlimods.arsmagicalegacy.api.magic.IManaHelper;
+import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMAttributes;
 import net.minecraft.world.effect.InstantenousMobEffect;
 import net.minecraft.world.effect.MobEffectCategory;
 import net.minecraft.world.entity.Entity;
@@ -16,6 +17,8 @@ public class InstantManaEffect extends InstantenousMobEffect {
     @Override
     public void applyInstantenousEffect(@Nullable Entity pSource, @Nullable Entity pIndirectSource, LivingEntity pLivingEntity, int pAmplifier, double pHealth) {
         IManaHelper manaHelper = ArsMagicaAPI.get().getManaHelper();
-        manaHelper.increaseMana(pLivingEntity, manaHelper.getMaxMana(pLivingEntity) / 5 * pAmplifier);
+        if (pLivingEntity.getAttributes().hasAttribute(AMAttributes.MAX_MANA.get())) {
+            manaHelper.increaseMana(pLivingEntity, manaHelper.getMaxMana(pLivingEntity) / 5 * pAmplifier);
+        }
     }
 }

